Pipfile

```diff
@@ -0,0 +1,20 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+pandas = "*"
+requests = "*"
+beautifulsoup4 = "*"
+selenium = "*"
+lxml = "*"
+
+[dev-packages]
```

Comment:
In the future this section will contain dependencies used by developers, like linters (flake8, pylint, etc), test frameworks (like pytest), and formatters (like black).